spec/support/seeds/seeds_e2e.rb

```diff
@@ -722,6 +722,48 @@
 
 ##############################################################################
 
+seeder.create_if_doesnt_exist(User, "email", "suspended-user@forem.local") do
```

Comment:
I needed the `suspend-user` and the `Suspended user article` for the Cypress specs.